features/admin.workflow-models.v1 /components/create/configuration-details-form.scss

```diff
@@ -0,0 +1,50 @@
+.flex-row-gap-30 {
```

Comment:
These kind of classes can be avoided by using the `<Grid/>` component in MUI/Oxygen UI - https://wso2.github.io/oxygen-ui/react/?path=/docs/layout-grid--overview